.github/workflows/build.yml

```diff
@@ -0,0 +1,78 @@
+name: build
+
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
+      - 'releases/*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - run: npm install
+    - run: npm run all
+    - name: check build up to date
+      run: git diff --exit-code --stat HEAD
+
+  test-e2e-all:
```

Comment:
The e2e tests under here operate on the source in dist/index.js. The build step verifies that file is kept up to date with src so we can be confident the generated code we're testing is correct.